app/models/landing_pages.rb

```diff
@@ -1,5 +1,8 @@
 class LandingPages
   def self.all
-    [ 'test-analytics' ]
+    [
+      "test-analytics",
```

Comment:
These are keys in `tiles.yaml`?

Reply:
Ah, these are identifier for pages, so Test Analytics and Integrations are Landing Pages as specified in the array.

By doing so it'll apply landing page behavior, such as removing the right column place to have a wider page layout to fit in the tiles.

As a side note, probably a bit confusing and not the best WX. I'll take a look at some WX improvements this Thursday (such as the ticket to supplant this behavior with `{:notoc}` or to not have YAML data everywhere).